spec/metrics/upstream_spec.lua

```diff
@@ -55,5 +55,19 @@ describe('upstream metrics', function()
         assert.stub(test_histogram.observe).was_not_called()
       end)
     end)
+
+
+    describe('With service id', function()
```

Comment:
These test only check the latency histogram. We should also add tests for the status code counter.